### ISSUE TITLE
Codecov submission is not enabled on Travis

### ISSUE THREAD

**Issue by chrisfilo** [nipy/nipype#1899]
It used to be, but it disappeared at some point. @djarecka do you know anything about it?

**Comment by satra** [comment]
we decided to remove it from one and just focus on the circle coverage. there didn't seem to be a way to accumulate coverage across travis + circle. since circle runs tests + workflows it seemed the better place to submit coverage reports from.

**Comment by chrisfilo** [comment]
I don't mind either way, but the whole point of CodeCov was to aggregate coverage reports across different CI providers. I'm surprised it didn't work. I don't think it's important enough to split hairs though.

**Comment by satra** [comment]
we won't split hairs, but we should check this out again.

**Comment by djarecka** [comment]
@chrisfilo  - yes, it was I who removed codecov from travis... I remember I did it after talking to @satra , but I don't recall that we concluded that it doesn't work. In fact in the first Pytest PR #1722 I used coverage from Travis and CircleCI.  I believe I wasn't sure how exactly CodeCov works and we simply agreed that CircleCI runs all tests from Travis anyway (plus some others), so we might just remove CodeCov from Travis, and then we're sure that the coverage is taken from CircleCI (and not from Travis only), but I might be wrong.
Can't really find any explanation in my commit messages, but I'm pretty sure that I was checking that the coverage didn't decrease after removing CodeCov from Travis.
